### ISSUE TITLE
Build broken due to broken ninja repository

### ISSUE THREAD

**Issue by RoyCurtis** [EssentialsX/Essentials#1126]
When attempting to build EssentialsXParent, it fails when building EssentialsX. [This is because the `supervisor-bukkit` dependency itself has a dependency on `exhaust-bukkit`](http://i.imgur.com/yWFuTcG.png).  [`exhaust-bukkit` is only available on a repository](https://github.com/SupaHam/SupaCommons/blob/master/commons-bukkit/pom.xml#L32) which is [currently returning HTTP error 500](http://repo.miserable.ninja/content/groups/public/).

# Reproduction

1. [Follow the Building instructions for EssentialsX](https://github.com/drtshock/Essentials#building)
2. When executing `mvn clean install`, observe the build of EssentialsX fails
    * [Maven log of build failure](https://gist.github.com/RoyCurtis/8ee9a730b972bcff80381fc4b9f46e31)

# Workaround

1. Clone https://github.com/MiserableNinja/Exhaust into a directory with `git clone https://github.com/MiserableNinja/Exhaust.git`
2. Inside the cloned Exhaust directory, execute `gradlew install`
3. Try building EssentialsXParent again

## Explaination

_This is for build system newbies, like I, who are confused about how Maven and Gradle work_

[The `install` gradle task is provided by the `maven` plugin](http://stackoverflow.com/questions/6122252/gradle-alternate-to-mvn-install), and therefore acts like a maven installation.

A maven installation is when a project, such as Exhaust, [is installed to your system's local maven repository](http://i.imgur.com/Ei4SJIU.png). Maven will then use this local repository (which can be seen as a cache) when looking for the `exhaust-bukkit` dependency, thus bypassing the broken ninja repository.

This also explains why running `java -jar BuildTools.jar --rev 1.8` etc. magically makes EssentialsX buildable; [BuildTools installs the CraftBukkit jars to the local system repository, too.](http://i.imgur.com/AJ4psLz.png)

# Notes

[This repository appears to have been broken since April 2016](https://github.com/MiserableNinja/Exhaust/issues/8). If I assumed correctly that the repository maintainer is @kashike, I have emailed them to notify of the repository issue.

**Comment by mdcfe** [comment]
I don't see how this can be fixed on EssentialsX's end without simply removing the Supervisor dependency - it's not a direct dependency, and as far as I can tell SupaHam's commons-minecraft only uses the repos it specifies in its own POM to find Exhaust.

**Comment by RoyCurtis** [comment]
@md678685: Indeed, it's not EssentialsX' fault per se. I opted to report it here however, as I imagine more people would be trying to build EssentialsX than commons-minecraft, and thus be running into this issue. I posted it for the workaround more than anything.

That said, I am closing this issue. It appears @kashike has brought the repository back up and I was able to build EssentialsX (after deleting `ninja.*` from my local `.m2` repository). Thank you very much, kashike!
